testdata/b3/ca.cert

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can you add a README.md to this directory, with instructions (e.g. shell commands) to generate these?

Reply:
Added